client/components/input-chrono/docs/example.jsx

```diff
@@ -17,43 +16,40 @@ import Card from 'components/card';
 /**
  * Date Picker Demo
  */
-const InputChronoExample = localize(
-	React.createClass( {
-		mixins: [ PureRenderMixin ],
-
-		getInitialState() {
-			return {
-				date: this.props.moment(),
-			};
-		},
-
-		componentWillMount() {
-			var self = this;
-			this.interval = setInterval( function() {
-				var date = self.props.moment( self.state.date );
-				date.hours( date.hours() + 1 );
-				self.setState( { date: date } );
-			}, 1000 );
-		},
-
-		componentWillUnmount() {
-			clearInterval( this.interval );
-		},
-
-		onSet( date ) {
-			console.log( `date: %s`, date.toDate() );
-			this.setState( { date: date } );
-		},
-
-		render() {
-			return (
-				<Card style={ { width: '300px', margin: 0 } }>
-					<InputChrono value={ this.state.date.calendar() } onSet={ this.onSet } />
-				</Card>
-			);
-		},
-	} )
-);
+const InputChronoExample = localize(/**
+ * Date Picker Demo
+ */
```

Comment:
Ugh, we should fix this. I'll push a commit.

Reply:
nice catch. it also occured in `client/components/post-schedule/docs/example.jsx`.